### PR TITLE
fix: use only filtered data for maps

### PR DESCRIPTION
**Pull request by mfshao**
Deployed in https://qa-covid19.planx-pla.net for testing

### Bug Fixes
- Use only `open-JHU` data to calculate `selectedDate`

### Improvements
- Pass filtered data to child map components in props

### CODE REVIEW

File: src/Covid19Dashboard/index.jsx [hunk 1]
```diff
@@ -215,7 +209,7 @@ class Covid19Dashboard extends React.Component {
                 />
               </div>
               <div className='covid19-dashboard_visualizations'>
-                <WorldMapChart {...this.props} />
+                <WorldMapChart rawMapData={this.props.rawData.filter(location => (location.project_id === 'open-JHU' && (location.confirmed.length || location.deaths.length)))} />
```
Comment (paulineribeyre) [inline src/Covid19Dashboard/index.jsx:212]: what do you need `&& (location.confirmed.length || location.deaths.length)` for? 😄

Comment (paulineribeyre) [reply src/Covid19Dashboard/index.jsx:212]: nvm, i see why. probably should also be checked for the illinois map, but whatever

Comment (mfshao) [reply src/Covid19Dashboard/index.jsx:212]: this logic https://github.com/uc-cdis/data-portal/pull/685/files#diff-891735d0760d4f7e2be89d9467b3ba0fL266 was filtering out no data locations. I'm just moving this into here

Comment (mfshao) [reply src/Covid19Dashboard/index.jsx:212]: > nvm, i see why. probably should also be checked for the illinois map, but whatever

i'll leave this task to you :smirk:

**Pull request merged**